README.md

```diff
@@ -23,40 +23,119 @@ $ npx @commercetools-frontend/create-mc-app my-new-custom-application-project --
 
 ## Developing application-kit packages
 
-Install the dependencies (uses yarn workspaces):
+### Initial Installation
+
+1. Clone this repo
+
+   ```bash
+   $ git clone https://github.com/commercetools/merchant-center-application-kit.git
+   ```
+
+2. Add necessary environment variables
+
+   > Navigate to `merchant_center_application_kit/playground`, duplicate `.env.local.template`, name the duplicate `.env.local` and add these values:
+
+   ```bash
+     MC_API_URL="https://mc-api.europe-west1.gcp.commercetools.com" # for prod
+     APP_ID="" # can be an empty string for dev
+     CTP_INITIAL_PROJECT_KEY=<your-project-name> # the name of any project you have access to on prod/stage
+   ```
+
+3. Build and run the application kit
+
+   > In a new terminal window, navigate to the project root directory - `/merchant_center_application_kit` - and run:
+
+   ```bash
+   $ yarn && yarn prebuild && yarn build && yarn build:watch
+   ```
+
+   > Confirm that the watch is running successfully, you should see:
+
+   ```bash
+   > "info @commercetools-frontend/application-shell waiting for changes..."
+   >...
+   ```
+
+4. Remove fake permission:
+   > `PERMISSIONS.ViewPlaygroundStateMachines` which is required by playground is a fake permission and you might need to remove it
+   > in order to be able to run and view the application. You'll need to remove it twice in `custom-application-config.mjs` and once in
+   > `routes.js` so that `permissions`/`demandedPermissions` are just empty objects:
+
+`custom-application-config.mjs`
```

Comment:
I'm not sure I understand this point. Why fake?

Reply:
![Screen Shot 2021-09-21 at 3 00 10 PM](https://user-images.githubusercontent.com/13600903/134233049-9e4144f0-6759-4a4b-83b9-c445541c4b69.png)

@emmenko 
"Fake" might not be the best terminology but without removing them in those two files you cannot access the playground (see screenshot). Fedor helped me discover this and he called them fake permissions which is why I used that term but I can change it if there's a better alternative. We were curious though why they were there in the first place and if they needed to be removed permanently.